.circleci/config.yml

```
@@ -22,10 +22,14 @@ jobs:
       - install_uv
       - run: uv sync --dev --no-python-downloads
       - run: uv lock --locked
+      - run: npm install
```

```suggestion
      - run: npm ci
```

to install from lockfile